src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -170,6 +171,12 @@ const AppFrame = ({ appUrl }: Props): ReactElement => {
       chainId: NETWORK_ID,
     }))
 
+    communicator?.on('getSafeBalances', async () => {
+      const balances = await fetchTokenCurrenciesBalances({ safeAddress, selectedCurrency: 'usd' })
```

Comment:
Can we pass the currency as a param from the Safe App and make USD the default value?

Reply:
We can, I'm just not really sure if we want to offer currency conversion API, I thought about it just as a bonus

Reply:
It could also make sense to return user's selected currency value

Reply:
True, the default should be probably the user's currency, not USD.
Reg. the param itself, I just suggested it because if `fetchTokenCurrenciesBalances` exposes it, why not expose it to safe apps?

Reply:
@katspaugh is there a list of available currencies? what format it should support?

Reply:
You can fetch this from the client gateway

Reply:
I added a currency param, please check again